### ISSUE TITLE
Provide summary details in API

### ISSUE THREAD

**Issue by SID800** [ssllabs/ssllabs-scan#533]
The text and numbers for each section provided in the summary, would be beneficial to have in available the API.

For the text, I'm talking about the stuff like: "This server is vulnerable to the POODLE TLS attack. Patching required. Grade set to F. "  so a user of the API doesnt have to enumerate through the many ways a site can fail various best practices -- or why the analyze results gave/capped a specific score.

for the numbers, the 0 to 100 scale for certificate, Protocol Support, key exchange, cipher strength would also be helpful to have access to.

**Comment by seccubus** [comment]
Yes, please. I've been asking for a similar thing for quite some time now.

**Comment by Jocdaj** [comment]
What is the status on this enhancement?
